tasks/managed/update-trusted-tasks/update-trusted-tasks.yaml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: update-trusted-tasks
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to update trusted-tasks list bundles.
+    Once a task is released as a bundle, the list of trusted-tasks should be 
+    updated with the new task.
+    If an OCI artifact named data-acceptable-bundles is already in place we will use
+    it as an input and append it, else we will create a new artifact
+
```

Comment:
Empty line - should probably be removed.

Reply:
Empty line removed